rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] in this class Getters and Setters for @book and @person instance variables should be present

Reply:
done